tests/nightly/nightly-autorun.sh

```diff
@@ -16,8 +16,10 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
+git clean -fd
```

Comment:
Yea, the secrets should definitely be in a different place.  On my system I have the secrets checked out in a separate folder that we should git fetch and pull as well so that we are running off the latest secrets always too.